### PR TITLE
Features/administrator deliverable

### PR DESCRIPTION
**Pull request by maggie97**

### CODE REVIEW

File: teckers/teckers/Models/Network/Router/BatchRouter.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
+//
+//  BatchRouter.swift
+//  teckers
+//
+//  Created by Maggie Mendez on 15/11/19.
+//  Copyright © 2019 Teckers. All rights reserved.
+//
+
+import Foundation
```
Comment (haydeeR) [inline teckers/teckers/Models/Network/Router/BatchRouter.swift:9]: This is just an advice you can create a base Router and then create BatchRouter, ProgramRouter as a child of BaseRouter

File: teckers/teckers/Models/Tecker.swift [hunk 2]
```diff
@@ -0,0 +1,9 @@
+//
```
Comment (haydeeR) [inline teckers/teckers/Models/Tecker.swift:1]: Try never commit empty files

File: teckers/teckers/Models/Tecker.swift [hunk 3]
```diff
@@ -5,5 +5,10 @@
 //  Created by Maggie Mendez on 16/12/19.
 //  Copyright © 2019 Teckers. All rights reserved.
 //
+import UIKit
 
-import Foundation
+protocol Tecker {
+    var name: String { get }
+    var teckerId: String { get}
+    var imageURL: String? {get}
```
Comment (haydeeR) [inline teckers/teckers/Models/Tecker.swift]: Try follow the same standard some between {get} and { get } not {get }

File: teckers/teckers/View/Deliverables.storyboard [hunk 4]
```diff
@@ -0,0 +1,333 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (haydeeR) [inline teckers/teckers/View/Deliverables.storyboard:1]: Storyboards are difficult to read, try to complement it with the commit message

**Review by haydeeR — APPROVED** [review]
I'd love a flowchart of your code specifically about your network
If you are available to do it, that would be great!

**Comment by haydeeR** [comment]
@maggie97  I'd like to you add a suit test cases to test Network layer.
If you agree I can create an issue for this.

**Comment by maggie97** [comment]
@haydeeR I'm agree with the test cases, but I don't know how do that. I will need to research about the test in iOS.  ^^
Also I will try to do the flowchart.

**Pull request merged**